src/Symfony/Bundle/WebProfilerBundle/Resources/config/toolbar.php

```diff
@@ -24,6 +24,7 @@
                 service('router')->ignoreOnInvalid(),
                 abstract_arg('paths that should be excluded from the AJAX requests shown in the toolbar'),
                 service('web_profiler.csp.handler'),
+                service('data_collector.dump'),
```

Comment:
please add `->ignoreOnInvalid()` to stay compatible with cases where VarDumper is not installed and configured by DebugBundle (and so the panel is not configured)

Reply:
Thanks for the fast review stof, really appreciate it.

I saw that the WebProfilerBundle has `symfony/http-kernel` as dependency and thought that service will be always there then. Adapted.